scripts/hf2bb.sl

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+# script version: slurm
+#
+# must be run with sbatch hf2bb.sl [realisationDirectory] [managementDBLocation]
+
+#SBATCH --cluster=Mahuika
```

Comment:
Probably wanna revise these slurm headers. Probably single core on mahuika.

Reply:
Does calc single rrup have 40 cores too - good to check while we are here?

Reply:
I've dropped the calc_rrups _single core count to 1.
Run times for AlpineF2K on Maui lander were lf2bb:32 mins, hf2bb: 3mins, so these hard coded times should be ok for everything